SPECS/kernel/config

```diff
@@ -7206,9 +7206,9 @@ CONFIG_DEBUG_INFO=y
 # CONFIG_DEBUG_INFO_SPLIT is not set
 CONFIG_DEBUG_INFO_DWARF_TOOLCHAIN_DEFAULT=y
 # CONFIG_DEBUG_INFO_DWARF4 is not set
-CONFIG_DEBUG_INFO_BTF=y
```

Comment:
Let's leave this one enabled. We would like to keep BTF type information available.